src/bio/bio_xstream.c

```diff
@@ -201,16 +201,26 @@ bio_nvme_init(const char *nvme_conf, int shm_id, int mem_size,
 		return -DER_INVAL;
 	}
 
-	bio_chk_cnt_max = (mem_size / tgt_nr) / size_mb;
-	D_INFO("Set per-xstream DMA buffer upper bound to %u %uMB chunks\n",
-	       bio_chk_cnt_max, size_mb);
+	d_getenv_bool("DAOS_SCM_RDMA_ENABLED", &bio_scm_rdma);
+	D_INFO("RDMA to SCM is %s\n", bio_scm_rdma ? "enabled" : "disabled");
+
+	if (is_nvme_configued) {
```

Comment:
This confuse me, if nvme conf file isn't passed in, we'll go through all following setup setps?